lib/command/info.rb

```diff
@@ -41,20 +41,8 @@ def call
 
     private
 
-    def app_matches?(app, app_name, app_options)
-      app == app_name.to_s || (app_options[:match_if_app_name_starts_with] && app.start_with?(app_name.to_s))
-    end
-
-    def find_app_options(app)
-      @app_options ||= {}
-      @app_options[app] ||= config.apps.find do |app_name, app_options|
-                              app_matches?(app, app_name, app_options)
-                            end&.last
-    end
-
     def find_workloads(app)
-      app_options = find_app_options(app)
-      return [] if app_options.nil?
```

Comment:
This line would never be executed, because this method is only called for apps that we're 100% sure are in `controlplane.yml`.